pio/isr/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+add_executable(isr)
```

Comment:
Looking at some of the other examples in https://github.com/raspberrypi/pico-examples/tree/master/pio it probably makes sense to use `pio_isr` in `CMakeLists.txt` rather than just `isr` ? :shrug: